bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -917,6 +936,7 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{
           <p class="title" itemprop="jobTitle">{{ _('VP, Product Marketing') }}</p>
 
           <ul class="utility">
+            <li><a class="file" rel="external" href="https://assets.mozilla.net/leadership/mozilla-lindsey-shepard.zip">{{ _('Download photos') }}</a></li>
```

Comment:
This file is 404ing. 😞

Reply:
Doh, looks like I missed one. Uploaded now.